packages/RatPack.package/RPApplication.class/instance/parseRequestContent..st

```diff
@@ -0,0 +1,12 @@
+private
+parseRequestContent: aHttpRequest
+
+	| method |
+	method := aHttpRequest method asLowercase asSymbol.
+	
+	[
+		(method = #put) ifTrue: [ self decodeContent: aHttpRequest rawPutContents type: aHttpRequest contentType. ].
+		(method = #post) ifTrue: [ self decodeContent: aHttpRequest rawPostContents type: aHttpRequest contentType. ].
+	] on: Error do: [ :sig |
+		self parsedRequestContents: nil.
+		self settings forceCorrectContentDecoding ifTrue: [ ^ sig pass ] ifFalse: [ ^ sig return ] ].
```

Comment:
I think if we are in debug mode, we should print a more descriptive error page which shows the error or try stop the server on error (see `RPApplication >> serverError`) to enable the developer to debug.

Reply:
In theory, this is an error that can only be triggered by a bad request, where serverError is something that is triggered by an error on the server side, so I think this is somewhat different. It may be helpful if you're trying to build new content decoders.

I think stopping the server and opening a debugger window is a bad idea, since this will cause every malformed request to stop the server. This leaves us only with the possibility of an error page.

This signal we pass here is propagated up (its caught in `processRequest`I think) where `badRequest` will be called if this happens. In my opinion, if we want to show a page that makes debugging easier, we should implement it in `badRequest`, just like it's implemented in `serverError`. This would require a rather complex new template, so I'd create a new issue for this. What do you think?

Reply:
See #75